test/test-ipcctl-notifications.js

```diff
@@ -1,274 +1,274 @@
-var agent = require('strong-agent');
```

Comment:
Don't modify the tests, you can just select which tests to run in the test script of package.json